assets/javascript/app.js

```diff
@@ -373,97 +417,96 @@ $(document).ready(() => {
 
-
-            //Update the page
-            $(resultsSelector).append(
-                "<button class='" + flightCat + "'>" + flightCat + "\xa0\xa0\xa0\xa0\xa0\xa0\xa0\xa0\xa0\xa0\xa0" + "</button>",
-                "<br>",
-                // " (VFR-Green, MVFR-Blue, IFR-Red, LIFR-Magenta)",
-                "<span class='station'>" + station + "</span>",
-                "<br>",
-                stationFullName,
-                "<br>",
-                "<span>Reported:</span> ",
-                // reportTime,
-                // " - ",
-                reportTimeFromNow,
-                "<br>",
-                "<span>Density Altitude:</span> ",
-                densityAlt, " ft",
-                "<br>",
-                "<span>Coordinates:</span> ",
-                coordinates,
-                "<br>",
-                "<span>Field Elevation:</span> ",
-                fieldElev,
-                " ft",
-                "<br>",
-                "<span>Temperature:</span> ",
-                tempC,
-                " &#8451;",
-                " - ",
-                tempF,
-                " &#8457;",
-                "<br>",
-                "<span>Dew Point:</span> ",
-                dewpointC,
-                " &#8451;",
-                " - ",
-                dewpointF,
-                " &#8451;",
-                "<br>",
-                "<span>Dew Point Spread:</span> ",
-                dewPointSpread,
-                " &#8457;",
-                "<br>",
-                "<span>Barometric Pressure:</span> ",
-                baro,
-                " inHg",
-                "<br>",
-                "<span>Wind Speed:</span> ",
-                windSpeed,
-                " mph",
-                "<br>",
-                "<span>Wind Direction:</span> ",
-                windDir,
-                " &deg;",
-                "<br>",
-                "<span>Wind Gust:</span> ",
-                windGust,
-                " mph",
-                "<br>",
-                "<span>Visiblility:</span> ",
-                visibility,
-                " sm",
-                "<br>",
-                "<span>Description:</span> ",
-                description,
-                "<br>",
-            );
+            let icon = getIcon(description);
+            console.log(icon);
+
+            $(resultsSelector + " .weatherIcon").html(icon);
+            $(resultsSelector + " .weatherDesc").html(description.toUpperCase());
+            $(resultsSelector + " .airportCode").html(station);
+            $(resultsSelector + " .airportFullName").html(stationFullName);
+            $(resultsSelector + " .timeStamp").html("Reported: " + reportTimeFromNow);
+            $(resultsSelector + " .flightCat").html("<button class='" + flightCat + "'>" + flightCat + "\xa0\xa0\xa0\xa0\xa0\xa0\xa0\xa0\xa0\xa0\xa0" + "</button>");
+            $(resultsSelector + " .compass").text("Compass here");
```

Comment:
@lmasullo This is an example code for `.compass` container. 
Please modify this line to show dynamic compass action. ( `.text();` wipes out the div - please remove it and replace with new codes)